pages/common/tsv-filter.md

```diff
@@ -0,0 +1,36 @@
+# tsv-filter
+
+> Filter lines of a TSV file by running tests against individual fields.
+> More information: <https://github.com/eBay/tsv-utils#tsv-filter>.
+
+- Print the lines where a specific column is numerically equal to a given number:
+
+`tsv-filter -H --eq {{field_name}}:{{number}} {{path/to/tsv_file}}`
+
+- Print the lines where a specific column is [eq]ual/[n]on [e]qual/[l]ess [t]han/[l]ess than or [e]qual/[g]reater [t]han/[g]reater than or [e]qual to a given number:
+
+`tsv-filter --{{eq|ne|lt|le|gt|ge}} {{column_number}}:{{number}} {{path/to/tsv_file}}`
+
+- Display the lines where a specific column is [eq]ual/[n]ot [e]qual/part of/not part of a string:
```

Comment:
```suggestion
- Print the lines where a specific column is [eq]ual/[n]ot [e]qual/part of/not part of a given string:
```

Reply:
I don't remember what we use `print` or `display`. I am not sure about this suggestion. ;)